doc/DeveloperGuide.md

```diff
@@ -88,6 +88,29 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename person
+
+**MSS**
+1. User requests to rename person
+2. User enters name of person to be renamed and new name of the person to be renamed.
+3. AddressBook renames the person <br>
+Use case ends.
+
+**Extensions**
+
+2a. The addressbook is empty
+
+> Use case ends
+
+2b. Name that user wants to update is the same as name of the person that user wants to rename
+
+> 2b1. AddressBook shows an error message <br>
+  Use case ends
+  
+2c. The name of the person that user wants to update is not in addressbook
+
+> Use case ends
```

Comment:
Should the use case end in this case? Would it be good user experience to get no feedback?